web/frontend/test/components/ResultsForm.test.js

```diff
@@ -0,0 +1,89 @@
+import { mount } from "@vue/test-utils";
+
+import ResultsForm from "@/components/LegalDocumentSearch/ResultsForm";
+
+describe("ResultsForm", () => {
+  const id = "99";
+  const sourceId = "2";
+  const sourceOrder = 1;
+  const searchResults = [
+    {
+      id,
+      sourceId,
+      sourceOrder,
+    },
+  ];
+  const selectedResult = id;
+  const added = { sourceRef: id };
+
+  it("triggers the add-doc event when clicked", async () => {
+    const wrapper = mount(ResultsForm, {
+      propsData: {
+        searchResults,
+      },
+    });
+    expect(wrapper.find("No legal documents were found").exists()).toBe(false);
+    await wrapper.find("li:nth-of-type(2)").trigger("click");
+    expect(wrapper.emitted("add-doc")).toBeTruthy();
+    expect(wrapper.emitted("add-doc")[0]).toEqual([id, sourceId]);
+  }),
+    it("does not allow submitting more than once", async () => {
```

Comment:
I think everything past that point is indented 1 step too deep?

Reply:
good 👁️ !